zerver/lib/send_email.py

```diff
@@ -380,6 +380,26 @@ def send_email_to_admins(
     )
 
 
+def send_email_to_billing_admins(
+    template_prefix: str,
+    realm: Realm,
+    from_name: Optional[str] = None,
+    from_address: Optional[str] = None,
+    language: Optional[str] = None,
+    context: Dict[str, Any] = {},
+) -> None:
+    admins = realm.get_human_billing_admin_users()
```

Comment:
Will this go to all realm owners?  It might be good for a comment to reflect that.

Reply:
Okay. Renamed this function from `send_email_to_billing_admins` to `send_email_to_billing_admins_and_realm_owners`

Also renamed `get_human_billing_admin_users` to `get_human_billing_admins_and_realm_owner_users`.